oxide-vpc/tests/integration_tests.rs

```diff
@@ -485,72 +488,100 @@ const UFT_LIMIT: Option<NonZeroU32> = NonZeroU32::new(16);
 const TCP_LIMIT: Option<NonZeroU32> = NonZeroU32::new(16);
 
 fn g1_cfg() -> VpcCfg {
+    let ip_cfg = IpCfg::DualStack {
+        ipv4: Ipv4Cfg {
+            vpc_subnet: "172.30.0.0/22".parse().unwrap(),
+            private_ip: "172.30.0.5".parse().unwrap(),
+            gateway_ip: "172.30.0.1".parse().unwrap(),
+            snat_cfg: Some(SNat4Cfg {
+                // NOTE: This is not a routable IP, but remember that a
+                // "public IP" for an Oxide guest could either be a
+                // public, routable IP or simply an IP on their wider LAN
+                // which the oxide Rack is simply a part of.
+                public_ip: "10.77.77.13".parse().unwrap(),
```

Comment:
Someone please notify past-Ryan that all IPs are routable, just a matter of how far that route can traverse. This makes me realize that `public_ip` was a poor choice I made long ago. We should probably rename this `external_ip` and remove this comment in the test.

Reply:
Renaming "public" to "external" seems like a good idea. Would you like me to do that now, or make another issue? In either case, I'll remove the comment.

Reply:
I'll leave it up to you on doing it in this PR vs. follow up.

Reply:
We'll do it live. I'll change these now, while there's a ton of API churn anyway.